templates/operations.ts.hbs

```diff
@@ -121,7 +121,8 @@
         {{/if}}
       }>,
       rawResponse?: boolean,
-    ): Promise<Response | {{getReturnTypeFromOperation this}}> {
+      returnTypeParam?: returnType
+    ): Promise<Response | Request | {{getReturnTypeFromOperation this}}> {
```

Comment:
`Request` and `Response` are browser globals, but we also execute in a node environment and use the ones provided by `cross-fetch`. The differences between browser and node `fetch` implementations can be subtle, so make sure that the types used here are accurate.